content/src/fieldConfig/business-formation.json

```diff
@@ -242,6 +262,23 @@
         "radioYesText": "Yes",
         "error": "Select 'Yes' or 'No'."
       },
+      "isVeteranNonprofit": {
+        "label": "Non-Profit Veteran Corporation",
+        "description": "Is your Domestic Non-Profit a `Veteran Corporation|veteran-corporation`?",
+        "radioNoText": "No",
+        "radioYesText": "Yes",
+        "error": "Select 'Yes' or 'No'.",
+        "domNpVetCorpReviewText": "**Yes**, my Domestic Non-Profit is a Veteran Corporation.",
+        "yesBoardMembersReviewText": "**Yes**, my Non-Profit will have board members.",
+        "noBoardMembersReviewText": "**No**, my Non-Profit will not have board members.",
+        "undefinedBoardMembersReviewText": "**My Non-Profit will or will not have board members**",
+        "boardMembersQualificationsReviewText": "**Board member qualifications are specified**",
+        "rightsAndLimitationsReviewText": "**Rights and limitations of the** board **members are specified**",
+        "choosingTrusteesReviewText": "**The method for choosing trustees are specified**",
+        "distributingAssetsReviewText": "**The method for distributing assets are specified**",
+        "inForm": "in this form.",
+        "inByLaws": "in the bylaws."
```

Comment:
[pebble] I see these are being used to construct the sentence on the review page

1. if something is used for review only, the key should indicate that
2. (overriding item 1) I don't think these are necessary. Why not just use the text we already have for `radioInFormText`, and apply a `toLowercase()` on it and add a period? That way, if the content team wants to change the text, they don't need to make the change in two places to keep it consistent.